sound/soc/sof/imx/imx8.c

```diff
@@ -372,7 +372,10 @@ static void imx8_ipc_msg_data(struct snd_sof_dev *sdev,
 			      struct snd_sof_pcm_stream *sps,
```

Comment:
Maybe you can update the commit message for both imx8 and imx8m (not just imx8m_ipc_msg_data).

Reply:
Will fix.